### ISSUE TITLE
Not all questions have titles

### ISSUE THREAD

**Issue by jordantomax** [AdmitHub/college-chooser#5]

**Comment by jordantomax** [comment]
i.e. class rank, gpa, range

**Comment by jordantomax** [comment]
also 'Gpa' should be GPA

**Comment by yourcelf** [comment]
The titles bit is a consequence of reusing the form flow logic for generating the answers list. The titles (or lack) are what show on the initial survey.

**Comment by jordantomax** [comment]
makes sense, probably a good idea to do a sweep through all of the questions to clean that stuff up. I'm gonna leave that to Andrew, because I think he has a better idea of what the copy should read.

**Comment by AndrewMagliozzi** [comment]
I'll handle all this stuff next week when this conference is over.

**Comment by jordantomax** [comment]
Cool, sounds good Drew!
